packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -89,6 +87,10 @@ export const BlockSwitcherDropdownMenu = ( { clientIds, blocks } ) => {
 		[ clientIds, blocks, blockInformation?.icon ]
 	);
 
+	const blockTitle = useBlockDisplayTitle( {
+		clientId: clientIds,
```

Comment:
This needs to be tested with multiple blocks selected. Previously, it would use the name of the first block with:

```js
const [ { name: firstBlockName } ] = blocks;
```

Reply:
You're right! I'll probably just have to pick the first one..